dove/tests/test_cmd_dove_run.rs

```diff
@@ -0,0 +1,65 @@
+#![cfg(test)]
```

Comment:
Those tests should actually check different flags for the `dove run`, specifically add `--signers` check.